### ISSUE TITLE
Inconsistencies between expected state for different API calls

### ISSUE THREAD

**Issue by jaschaio** [prescottprue/redux-firestore#202]
I have noticed some inconsistencies between api calls. If I use `store.firestore.setListener()` the `firestore.status.received`, `firestore.status.requested` and `firestore.status.timestamp` values are populated.

If I use instead `store.firestore.get()` none of those fields is populated.

On the first sight I can't really see why. If the `firestore.status` state slice should be used for loading indicators, it should be consisten across all api calls that request data from firestore, no matter if I am just requesting it once or setting up a listener.

**Comment by prescottprue** [comment]
Yeah I agree it should be consistent accross API calls - I started by just making the actions dispatch and had planned to getting to the reducer logic. Open to a PR if you get a chance.
